Sources/Controllers/MyPlaces/TracksViewController.swift

```diff
@@ -490,6 +510,83 @@ final class TracksViewController: OACompoundViewController, UITableViewDelegate,
         filterButton.configuration = currentConfig
     }
     
+    private func createSortMenu() -> UIMenu {
```

Comment:
Move all the logic (functions) related to sorting into a separate extension. Can keep it in the same class